taskcluster/scripts/decision-install-sdk.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -ex
+
+export CURL='curl --location --retry 5'
+
+ANDROID_SDK_VERSION='3859397'
```

Comment:
We probably want to centralize this value (and its checksum) in a single file, in a followup. I didn't tackle it here because the split has been there so far, and because it introduces more complexity.

Reply:
I suspect that we want to use a toolchain here too, somehow (need to spend some time thinking about bootstrapping issues with that though[1]). I think that will handle centralizing this value.

[1] One idea I had was to handle that as part of refactoing things so there is a shared mobile-taskgraph repository, that defines these toolchains.